cmd/storage-rest-common.go

```diff
@@ -41,7 +41,6 @@ const (
 	storageRESTMethodDeleteFileBulk = "deletefilebulk"
 	storageRESTMethodRenameFile     = "renamefile"
 	storageRESTMethodVerifyFile     = "verifyfile"
-	storageRESTMethodGetInstanceID  = "getinstanceid"
```

Comment:
you need to bump the storage version here since you are changing the API @krishnasrinivas